web/modules/custom/joinup_core/src/JoinupRelationManager.php

```diff
@@ -35,6 +36,13 @@ class JoinupRelationManager implements JoinupRelationManagerInterface, Container
    */
   protected $entityTypeManager;
 
+  /**
+   * The sparql entity storage.
+   *
+   * @var \Drupal\sparql_entity_storage\SparqlEntityStorageInterface
+   */
+  protected $sparqlStorage;
```

Comment:
This static cache should be removed, see the comment below at `::getRdfStorage()` for a motivation why this is not a good idea.